en/docs/references/preview-features.md

```diff
@@ -0,0 +1,16 @@
+# Preview Features
+
+Preview features offer optional functionality that you can enable if required.
+
+!!! info
+    Preview features cannot be enabled when you are logged in anonymously.   
```

Comment:
Should we rephrase:
"You cannot enable preview features when you try Choreo anonymously."